backend/frontend/tests/test_all_claims_page.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase, Client
```

Comment:
We are making use of Playwright for UI testing so let's make sure we are consistent.

Have a look at the other frontend tests to see how we handle the login and asserting if things are on the page or not. Eg. make use of `FrontendMixin`, etc.